manifests/a/ArduinoSA/IDE/stable/2.3.4/ArduinoSA.IDE.stable.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created with komac v2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ArduinoSA.IDE.stable
+PackageVersion: 2.3.4
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Custom: ALLUSERS=1
+UpgradeBehavior: install
+FileExtensions:
+- c
+- c++
+- cc
+- cpp
+- cxx
+- h
+- h++
+- hh
+- hpp
+- hxx
+- i++
+- inc
+- ino
+- ipp
+- ixx
+ProductCode: '{C42CE043-BA31-4C55-AED2-8F0563216C04}'
+ReleaseDate: 2024-12-03
+AppsAndFeaturesEntries:
+- DisplayName: Arduino IDE
+  Publisher: Arduino SA
+  DisplayVersion: 2.3.4.0

```

Comment:
```suggestion
```
`DisplayVersion` isn't necessary here, as the package manager adds trailing zeroes.